### PR TITLE
Bug 1145842 - Include J2ME emulator in build. r=kgrandon

### PR DESCRIPTION
**Pull request by DouglasSherk**

### CODE REVIEW

File: getj2me [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+node --harmony ./_getj2me
```
Comment (KevinGrandon) [inline getj2me:13]: Can't we just rename `_getj2me` to `getj2me` and execute that file directly? What's the reasoning for having this as a runner for it? If we don't want to get rid of this, can you add a comment in this file as to why we need it?

File: _getj2me [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+var https = require('https');
+var exec = require('child_process').exec;
+var spawn = require('child_process').spawn;
+var url = require('url');
+var fs = require('fs');
+
+var package;
+
+var begin = function() {
+  console.log(PREFIX, 'Starting app download process.');
```
Comment (KevinGrandon) [inline _getj2me:18]: nit: I would opt to just use the node debug module which should handle the prefixes for you?

Comment (DouglasSherk) [inline _getj2me:18]: Sure, in the future, I will.

File: getj2me (hunk 1 above)
Comment (DouglasSherk) [inline getj2me:13]: This is needed. See https://github.com/fxos/customizer/pull/11

I'll add a comment explaining it.

**Pull request merged**